packages/core/src/components/ChoiceList/Choice.example.jsx

```diff
@@ -20,13 +40,45 @@ ReactDOM.render(
     </Choice>
 
     <div className="ds-u-margin-top--4">
-      <Choice defaultChecked name="radio_choice" type="radio" value="a">
+      <fieldset className="ds-c-fieldset">
+        <legend className="ds-c-label">Checkboxes with children</legend>
+        <Choice defaultChecked name="radio_choice" value="a">
+          Checkbox A
+        </Choice>
+        <Choice name="radio_choice" value="b" checkedChildren={childSelect}>
+          Checkbox B - with children
+        </Choice>
+        <Choice name="checkbox_choice" value="c">
+          Checkbox C
+        </Choice>
+      </fieldset>
+    </div>
+
+    <div className="ds-u-margin-top--4">
```

Comment:
We will want to wrap each one of these groupings in a `<fieldset>` with a proper `<legend>` and update the name attributes to match on each group. This will help prevent errors when we add an e2e test that runs axe on this component.

Reply:
Updated